assets/js/app.js

```diff
@@ -11,6 +11,12 @@ $(function() {
   var settings = get_settings();
   var access_token = localStorage.getItem("access_token");
   var particle = new Particle();
+  var varTypeColors = {
+    bool: 'primary',
+    int: 'warning',
+    double: 'danger',
+    string: 'success'
```

Comment:
Can regular color names be used instead? Would be easier for inexperienced users to understand or change.